config/systems-uenv/clariden-uenv.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2016-2023 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# ReFrame CSCS settings
+#
+
+import os
+import pathlib
+import yaml
+
+from reframe.core.exceptions import ConfigError
+
+uenv = os.environ.get('UENV', None)
+
+if uenv is None:
+    raise ConfigError('UENV is not set')
+
+# FIXME: Only the first image:mount pair is currenty used
+uenv_list = uenv.split(',')
+uenv_first = uenv_list[0]
+
+uenv_file, *image_mount = uenv_first.split(':')
+if len(image_mount) > 0:
+    image_mount = image_mount[0]
+else:
+    image_mount = '/user-environment'
+
+image_path = pathlib.Path(uenv_file)
+if not image_path.exists():
+    raise ConfigError(f"uenv image: '{image_path}' does not exist")
+
+image_name = image_path.stem
+
+# Options for the Slurm plugin to mount the Squashfs uenv image
+uenv_access = [f'--uenv-file={uenv_file} --uenv-mount={image_mount}']
+
+try:
+    rfm_meta = image_path.parent / f'{image_name}.yaml'
+    with open(rfm_meta) as image_envs:
+        image_environments = yaml.load(
+            image_envs.read(), Loader=yaml.BaseLoader)
+except OSError as err:
+    raise ConfigError(f"problem loading the metadata from '{rfm_meta}'")
+
+
+environs = image_environments.keys()
+environ_names =  ([f'{image_name}_{e}'for e in environs] or
+                  [f'{image_name}_builtin'])
+
+partitions = [
+    {
+        'name': f'nvgpu',
```

Comment:
```suggestion
        'name': 'nvgpu',
```
Not sure why you use f-strings?